### ISSUE TITLE
About the training loss

### ISSUE THREAD

**Issue by awesomeNabi** [softrepa/SoftREPA#1]
First of all, thank you for your outstanding work — I find the idea behind this project truly impressive.

I have a question regarding the training process of the sd3_dc model. I observed that the loss remains almost constant throughout the entire training. Is this expected behavior? If so, how should one properly evaluate the quality or progress of the training in this case?

Thank you in advance for your time and assistance.

**Comment by awesomeNabi** [comment]
Additionally, if possible, could you please explain why the loss might remain unchanged during training? I would greatly appreciate any insights into the underlying reasons or mechanisms behind this phenomenon.

<img width="1204" height="568" alt="Image" src="https://github.com/user-attachments/assets/bfdcfc2d-43e4-424b-ab92-57d08666ad3c" />
